<launch>
 <!-- Turtlesim Node-->
  <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <!-- <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/> -->
  <node pkg="mai_sim" type="joy_trajectory3" name="teleop"/>
</launch>

<!-->
$ roscore
$ rosparam set joy_node/dev "/dev/input/jsX"
$ rosrun joy joy_node
$ roslaunch mai_sim joy_trajectory.launch
<-->